mis_builder_cash_flow/report/mis_cash_flow.py

```diff
@@ -97,23 +96,22 @@ def init(self):
                     THEN -fl.balance
                     ELSE 0.0
                 END AS credit,
-                Null as reconciled,
-                Null as full_reconcile_id,
+                NULL as reconciled,
+                NULL as full_reconcile_id,
                 fl.partner_id as partner_id,
                 fl.company_id as company_id,
-                %i as user_type_id,
+                {} as user_type_id,
                 fl.name as name,
                 fl.date as date
             FROM mis_cash_flow_forecast_line as fl
-        """
-            % account_type_receivable.id
+        """.format(
```

Comment:
Just to know, why this change? I used to believe that in odoo the "standard" is to use % instead of format. Perhaps I was wrong and that standard is only for translatable strings.

Reply:
First, there was a `%d`. the documentation in https://www.psycopg.org/docs/usage.html says:

```
The variables placeholder must always be a %s, even if a different placeholder (such as a %d for integers or %f for floats) may look more appropriate:
```

Second, about using `%` or `format`. Both things are not recommended to be used because they may lead to sql-injection problems. That's why pylint has a sql-injection checker. Just happens, in this case, there is no problem of sql-injection, that's why there is no problem if we use `%` or `format`. I just preferred to use `format` as a saveproof, to add `NULL` in case there is no account_type_receivable.

Reply:
Please use `sql.SQL`